tests/foreman/ui/test_gpgkey.py

```diff
@@ -3,23 +3,27 @@
 
 from ddt import ddt
 from fauxfactory import gen_string
+from nailgun import client
 from robottelo import entities
+from robottelo.common import conf
 from robottelo.common.constants import (
     FAKE_1_YUM_REPO,
     FAKE_2_YUM_REPO,
     REPO_DISCOVERY_URL,
     VALID_GPG_KEY_BETA_FILE,
     VALID_GPG_KEY_FILE,
+    ZOO_CUSTOM_GPG_KEY
```

Comment:
Nitpick: add trailing comma. Doing so makes it easier to add, rearrange and remove lines without mucking with commas. It may also make future diffs more concise.